### PR TITLE
Restructure less files

### PR DESCRIPTION
**Pull request by SMahil**
#2138

### CODE REVIEW

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2139-1 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2139-fabric8-planner.badger.fabric8.io

File: src/app/components/iterations-panel/iterations-panel.component.html [hunk 1]
```diff
@@ -1,57 +1,74 @@
...
           Active Iteration
         </h3>
       </div>
-      <ul class="iteration-expanded-div">
-        <li class="active iteration-container" *ngFor="let iteration of activeIterations" [routerLinkActive]="'selected-iteration'"
+      <ul class="f8-itr__List--expanded">
+        <li class="active f8-itr__container"
+          *ngFor="let iteration of activeIterations"
+          [routerLinkActive]="'io-itr--selected'"
           [routerLink]="[]" [attr.data-id]="iteration.id"
```
Comment (nimishamukherjee) [inline src/app/components/iterations-panel/iterations-panel.component.html]: The selected active iteration does not get highlighted. The iteration gets highlighted in the tree view also.
Check what happens when you click on Sprint 137 in https://openshift.io/openshiftio/openshiftio/plan
iteration

File: src/app/components/iterations-modal/iterations-modal.component.html [hunk 2]
```diff
@@ -1,11 +1,12 @@
 <modal
+  class="f8-iteration-modal"
   #createUpdateIterationDialog
```
Comment (nimishamukherjee) [inline src/app/components/iterations-modal/iterations-modal.component.html]: The modal title is not seen. Also, the active OFF text is not seen.
createiteration

**Comment by SMahil** [comment]
#2140

**Pull request closed without merging**